### PR TITLE
UserExpenseChart | Pending :o: 🇺🇦

### PR DESCRIPTION
**Pull request by mediano11**
Refactor: 
- Replaced the output method of the title of the total amount of expenses in UserExpenseChart with chartJS canvas.
- Remade Circle element

Done: 
- Shadow Plugin for UserExpenseChart

Fix:
- Fixed some styles & variables

Add: 
- Added method for formatting numbers 
- Added parser for colours

### CODE REVIEW

File: src/store/UI_store/ThemeSlice/ThemeSlice.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const ThemeSlice = createSlice({
             initialState.theme = (initialState.theme === 'light') ? 'dark' : 'light'
             initialState.cardBackgroundColor = (initialState.theme === 'light') ? '0F0F0F': '#FFFFFF'
             initialState.backgroundColor = (initialState.theme === 'light') ? '#151515' : '#F8F8FF'
-            initialState.mainTextColor = (initialState.theme === 'light') ? '#EAEAEA': '#333333'
+            initialState.mainTextColor = (initialState.theme === 'light') ? '#333333' : '#EAEAEA'
             initialState.textColor = (initialState.theme === 'light') ? '#7A7A9D' : '#9BABC5'
```
Comment (tema-adidas) [inline src/store/UI_store/ThemeSlice/ThemeSlice.ts:27]: I guess that`s reversed text color order too🙃

Comment (mediano11) [reply src/store/UI_store/ThemeSlice/ThemeSlice.ts:27]: you are wrong

Comment (tema-adidas) [reply src/store/UI_store/ThemeSlice/ThemeSlice.ts:27]: 27 line?

Comment (mediano11) [reply src/store/UI_store/ThemeSlice/ThemeSlice.ts:27]: image

**Pull request merged**